libvast/vast/concept/parseable/string/quoted_string.hpp

```diff
@@ -27,13 +27,16 @@ class quoted_string_parser : public parser<quoted_string_parser<Quote, Esc>> {
 public:
   using attribute = std::string;
 
+  // clang-format off
   static constexpr auto esc = ignore(parsers::ch<Esc>);
   static constexpr auto quote = ignore(parsers::ch<Quote>);
-  static constexpr auto esc_esc = esc >> parsers::ch<Esc>;
   static constexpr auto esc_quote = esc >> parsers::ch<Quote>;
   static constexpr auto str_chr
-    = esc_esc | esc_quote | (parsers::print - quote);
+    = (parsers::print - quote - parsers::ch<Esc>)
```

Comment:
can we add `static constexpr auto esc = parsers::ch<Esc>;` and rename the old `esc` to `ignore_esc`?